### PR TITLE
Update test ui test_contenthost to use REX

### PR DESCRIPTION
**Pull request by swadeley**
Hello

Updating more tests in  /ui/test_contenthost.py  to use REX

Removed redundant `test_actions_katello_host_package_update_timeout` as its only relevant to katello-agent test.

------

I need to wait for this bugzilla to be resolved before I can test this file:
[Bug 1983043 - undefined method `prune_known_hosts!` when executing a remote job](https://bugzilla.redhat.com/show_bug.cgi?id=1983043)

Also
[1986554 – When trying to enable a module stream, 'unexpected token at "' error is displayed](https://bugzilla.redhat.com/show_bug.cgi?id=1986554)

EDIT: those are now verified. 
So now its just https://github.com/SatelliteQE/airgun/pull/607 that this PR needs.

### CODE REVIEW

**Comment by JacobCallahan** [comment]
@swadeley I see that the first bug has been verified and the second is modified. Could you still have this in and skip on the status of the second bug?

**Comment by swadeley** [comment]
> @swadeley I see that the first bug has been verified and the second is modified. Could you still have this in and skip on the status of the second bug?

In theory, but if the fix will be in the next snap, I will think about that.

**Comment by swadeley** [comment]
Hello

this PR depends on:
[ Add REX support to chost and errata tests #607 ](https://github.com/SatelliteQE/airgun/pull/607)

**Comment by swadeley** [comment]
Hello

some test that now use REX

robottelo]$ date && pytest tests/foreman/ui/test_contenthost.py -k test_positive_end_to_end_bulk_update -k package
Fri 20 Aug 12:27:57 BST 2021
============================== test session starts ====================================================
platform linux -- Python 3.9.6, pytest-6.2.4, py-1.10.0, pluggy-0.13.1
shared_function enabled - OFF - scope:  - storage: file
rootdir: /sat6/robottelo_1/robottelo, configfile: pyproject.toml
plugins: xdist-2.3.0, mock-3.6.1, ibutsu-1.16, forked-1.3.0, services-2.2.1, cov-2.12.1, reportportal-5.0.8
collected 28 items / 22 deselected / 6 selected                                                                                                                                                   

tests/foreman/ui/test_contenthost.py ......                                                                                                                                                 [100%]


========================== 6 passed, 22 deselected, 5 warnings in 4108.49s (1:08:28) ========================

**Comment by swadeley** [comment]
Hello

with [ Add REX support to chost and errata tests #607 ](https://github.com/SatelliteQE/airgun/pull/607),

all tests except two non REX tests in `tests/foreman/ui/test_contenthost.py` pass.

This one because of [Bug 1995228 - Applying errata from an Erratum's Content Host tab fails ](https://bugzilla.redhat.com/show_bug.cgi?id=1995228):
`tests/foreman/ui/test_contenthost.py::test_positive_end_to_end`

and this one because of unrelated change:
`tests/foreman/ui/test_contenthost.py::test_content_access_after_stopped_foreman`

File: tests/foreman/cli/test_satellitesync.py-test_new_positive_export_custom_product [hunk 1]
```diff
@@ -0,0 +1,2467 @@
+"""Test class for InterSatellite Sync
```
Comment (mshriver) [inline tests/foreman/cli/test_satellitesync.py-test_new_positive_export_custom_product:1]: It looks like you may have duplicated a test module here?

**Review by mshriver — CHANGES_REQUESTED** [review]
Changes to test_contenthost look good, but it looks like there is some extraneous content in the PR.

File: tests/foreman/ui/test_contenthost.py [hunk 2]
```diff
@@ -224,16 +226,17 @@ def test_positive_end_to_end(session, repos_collection, vm):
...
         # Ensure package installed
         packages = session.contenthost.search_package(vm.hostname, FAKE_0_CUSTOM_PACKAGE_NAME)
         assert packages[0]['Installed Package'] == FAKE_0_CUSTOM_PACKAGE
-        # Install errata
-        result = session.errata.install(FAKE_2_ERRATA_ID, vm.hostname)
-        assert result['result'] == 'success'
-        # Ensure errata installed
-        packages = session.contenthost.search_package(vm.hostname, FAKE_2_CUSTOM_PACKAGE_NAME)
-        assert packages[0]['Installed Package'] == FAKE_2_CUSTOM_PACKAGE
+        if not is_open('BZ:1995228'):
```
[the remark this answers is not in the document]
Comment (mshriver) [reply tests/foreman/ui/test_contenthost.py]: @swadeley I think you mean robottelo-runner (PRT), not PIT here (Product integration testing).

PRT can indeed run UI tests, it has the same selenium config as our core CI pipelines.

**Review by mshriver — APPROVED** [review]

File: tests/foreman/ui/test_contenthost.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (swadeley) [reply tests/foreman/ui/test_contenthost.py]: PRT = Pull Request Tester , thanks for the info

**Comment by swadeley** [comment]
Needs another rebase

**Comment by swadeley** [comment]
trigger: test-robottelo
pytest:  tests/foreman/ui/test_contenthost.py

**Review by latran — APPROVED** [review]
ack

**Comment by swadeley** [comment]
Thank you

**Pull request merged**